app/main.py

```diff
@@ -1,7 +1,17 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
 def get_weather() -> None:
     # write your code here
-    pass
+    URL = "http://api.weatherapi.com/v1/current.json?"
+    FILTERING = "Paris"
```

Comment:
According to the checklist, constants should be defined using uppercase style. Ensure that `URL` and `FILTERING` are defined in uppercase, which is correct here. However, make sure to maintain this style consistently throughout your code.